kurtosis-devnet/pkg/build/contracts.go

```diff
@@ -0,0 +1,101 @@
+package build
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"os/exec"
+	"text/template"
+)
+
+// ContractBuilder handles building smart contracts using just commands
+type ContractBuilder struct {
```

Comment:
What are these Go packages being used for? I don't see them being used in the Justfile above.

Reply:
Yeah, they'll be used when I get to the PR that introduces https://github.com/ethereum-optimism/optimism/blob/sigma/kurtosis-devnet-rebase/kurtosis-devnet/cmd/main.go

I wish stacked PRs were a thing on Github :(

Reply:
I see, ok.